### ISSUE TITLE
Should set up the _kSingleLineVerticalPadding in the SnackBar

### ISSUE THREAD

**Issue by realcarlos** [flutter/flutter#15782]
It is not very friendly to set these const value
```dart
const double _kSnackBarPadding = 24.0;
const double _kSingleLineVerticalPadding = 16.0;
const Color _kSnackBackground = const Color(0xFF323232);
```
because the content of `SnackBar` is a widget, we need to keep the background color same.
and if set content = a container , `24` and `16` are too big number.

**Comment by chinmaygarde** [comment]
I am having trouble understanding the issue. Can you elaborate with a reduced test case detailing what you are trying to achieve?

**Comment by realcarlos** [comment]
Ok , for example I want to set a container with borderRadius to the SnackBar ,I wish the backgroudColor is null , but now I could not do it .

<img width="376" alt="2018-03-23 12 15 44" src="https://user-images.githubusercontent.com/13853/37783116-9f5d6bfa-2e2f-11e8-9a6f-47495d82e3d3.png">

**Comment by YeungKC** [comment]
This looks to have been addressed by #61180 and #31275.
If anyone has any objections I will reopen this issue.
